headless/src/jsMain/kotlin/dev/fritz2/headless/foundation/portalling.kt

```diff
@@ -92,7 +96,7 @@ fun <C : HTMLElement> RenderContext.portal(
         PortalContainer(
             classes = classes,
             id = portalId,
-            scope = scope,
+            scope = scope + this.scope,
```

Comment:
Same as above: The sequence would override values from the `portal`-call by values from outer, exsiting scope.